reactor-netty-core/src/main/java/reactor/netty/ReactorNetty.java

```diff
@@ -335,7 +334,7 @@ static void addHandlerAfterReactorCodecs(Connection context, String
 
 	static boolean mustChunkFileTransfer(Connection c, Path file) {
 		// if channel multiplexing a parent channel as an http2 stream
-		if (c.channel().parent() != null && c.channel().parent().pipeline().get(Http2ConnectionHandler.class) != null) {
+		if (c.channel().parent() != null && c.channel().parent().pipeline().get(NettyPipeline.H2MultiplexHandler) != null) {
```

Comment:
isn't that an unrelated change?

Reply:
`reactor-netty-core` does not have dependency to `netty-codec-http2`
this change removes the usage of `Http2ConnectionHandler` which is in `netty-codec-http2`